README.md

```diff
@@ -37,6 +47,22 @@ public class Status {
         return this.value;
     }
 
+    /**
+     * returns true if given enum is part of the declared values.use it in your application to detect when values coming from outside of the app are not yet part of the declared values (i.e.: there is a new version of the enum that your application is not yet aware of.
+     *
+     */
+    public static Boolean isDeclaredValue(Status val) {
```

Comment:
This is redundant since users have access with `Status.declaredValues.contains(...)`. APIs should be follow sufficiency and completeness. Here, we are complete without the convenience boolean accessors.

Reply:
If we really want it, it should be at least as an object member:

`myStatus.isDeclaredValue()` or, maybe better, since we usually want to track not declared value:

`myStatus.isUnknown()`

Reply:
done